index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio </title>
+   <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
+    <link rel="stylesheet" href="style.css">
+  </head>
+      <body>
+      <div class="flex-container">
+        <div class="item1">
+          <header>
+            <p class="time"> 9:14 <img class="net" src="C:\Users\Harrison\phone network image.jpg" alt="phone network"></p>
+            <img class="logo" src="C:\Users\Harrison\Downloads\My Logo.png" alt="logo">
+            <img class="navicon" src="https://assets.stickpng.com/images/588a64c6d06f6719692a2d0c.png" alt="nav icon"><br>
+         
+          </header>
+        </div>
+        <div class="item2">
+          <div class="design">
+            <img class="ellipse11" src="file:///C:/Users/Harrison/Downloads/Ellipse%2011%20(1).svg" alt="ellipse 11">
+            <img class="rectangle4" src="C:\Users\Harrison\Downloads\Rectangle 4.svg" alt="rectangle4">
+            <img class="ellipse1" src="file:///C:/Users/Harrison/Downloads/Ellipse%203.svg" alt="ellipse1">
+            <img class="ellipse5" src="C:\Users\Harrison\Downloads\Ellipse 5.svg" alt="ellipse 5">
+            <img class="rectangle7" src="C:\Users\Harrison\Downloads\Rectangle 7.svg" alt="rectangle7">
+            <img class="ellipse8" src="C:\Users\Harrison\Downloads\Ellipse 8.svg" alt="ellipse8">
+            <img class="ellipse13" src="C:\Users\Harrison\Downloads\Ellipse 13.svg" alt="ellipse13">
+            <img class="ellipse12" src="C:\Users\Harrison\Downloads\Ellipse 12.svg" alt="ellipse12">
+            </div>
```

Comment:
- [ ] I think it would better if you add a background image in CSS file instead of adding img into your HTML file.its good for the code cleanness and better applying the background image. and also Please make sure that the background image is runddering properly.
### Your Design👇
![missing3333](https://user-images.githubusercontent.com/66992901/198217113-fdf35a41-9094-4157-9da7-e173547240f7.PNG)
### Use the Figma background template👇
![header-shapes mobile](https://user-images.githubusercontent.com/66992901/198218680-01a4bf79-49e0-4ca5-b5a4-363c351466a5.png)